modifié :         ifs-cars-server/src/main/java/fr/uge/corp/ifscars/…

alopesmendes opened the pull request:
…cars/Car.java

	modifié :         ifs-cars-server/src/main/java/fr/uge/corp/ifscars/cars/ICar.java
	modifié :         ifs-cars-server/src/main/java/fr/uge/corp/ifscars/cars/Storage.java
	modifié :         ifs-cars-server/src/main/java/fr/uge/corp/ifscars/command/Command.java
	modifié :         ifs-cars-server/src/main/java/fr/uge/corp/ifscars/rating/IRating.java
	modifié :         ifs-cars-server/src/main/java/fr/uge/corp/ifscars/renting/IRentingService.java
	modifié :         ifs-cars-server/src/main/java/fr/uge/corp/ifscars/renting/RentingService.java
	modifié :         renting-app/src/main/java/fr/uge/corp/ifscars/cars/ICar.java
	modifié :         renting-app/src/main/java/fr/uge/corp/ifscars/rating/IRating.java
	modifié :         renting-app/src/main/java/fr/uge/corp/ifscars/rating/Rating.java
	modifié :         renting-app/src/main/java/fr/uge/corp/ifscars/renting/IRentingService.java
	modifié :         renting-app/src/main/java/fr/uge/corp/rentingapp/command/Command.java

alopesmendes committed:
modifié :         ifs-cars-server/src/main/java/fr/uge/corp/ifscars/cars/Car.java
	modifié :         ifs-cars-server/src/main/java/fr/uge/corp/ifscars/cars/ICar.java
	modifié :         ifs-cars-server/src/main/java/fr/uge/corp/ifscars/cars/Storage.java
	modifié :         ifs-cars-server/src/main/java/fr/uge/corp/ifscars/command/Command.java
	modifié :         ifs-cars-server/src/main/java/fr/uge/corp/ifscars/rating/IRating.java
	modifié :         ifs-cars-server/src/main/java/fr/uge/corp/ifscars/renting/IRentingService.java
	modifié :         ifs-cars-server/src/main/java/fr/uge/corp/ifscars/renting/RentingService.java
	modifié :         renting-app/src/main/java/fr/uge/corp/ifscars/cars/ICar.java
	modifié :         renting-app/src/main/java/fr/uge/corp/ifscars/rating/IRating.java
	modifié :         renting-app/src/main/java/fr/uge/corp/ifscars/rating/Rating.java
	modifié :         renting-app/src/main/java/fr/uge/corp/ifscars/renting/IRentingService.java
	modifié :         renting-app/src/main/java/fr/uge/corp/rentingapp/command/Command.java

Alex1304 commented on ifs-cars-server/src/main/java/fr/uge/corp/ifscars/renting/IRentingService.java:
```diff
@@ -70,6 +70,17 @@
+	 * @return
+	 * @throws RemoteException
+	 */
+	String displayAverageRating(String model, long id) throws RemoteException;
```
il faudrait que ça renvoie un double, l'affichage c'est censé être le rôle du client. Côté serveur il faut manipuler des données brutes. Pour le type tu peux laisser `int` pour `IRating.getRating()`, mais pour la moyenne il faut bien un double.

alopesmendes replied:
J'ai modifie.

Alex1304 reviewed (changes requested):
Juste un truc à modifier sinon c'est bon

alopesmendes committed:
modifié :         ifs-cars-server/src/main/java/fr/uge/corp/ifscars/renting/IRentingService.java
	modifié :         ifs-cars-server/src/main/java/fr/uge/corp/ifscars/renting/RentingService.java
	modifié :         renting-app/src/main/java/fr/uge/corp/ifscars/renting/IRentingService.java
